controller_manager_msgs/srv/ListControllers.srv

```diff
@@ -3,3 +3,4 @@
 
 ---
 controller_manager_msgs/ControllerState[] controller
+controller_manager_msgs/ControllerState[] controller_group
```

Comment:
Sorry for being picky, but do we really need this? Do we need to differantiate between `controller` and `controller_group`? What is what?

Could you also add a bit of doc about fields in the message? (see for example "switch controller" service)